api/v1/openapi.yaml

```diff
@@ -2500,6 +2503,22 @@ definitions:
       snat-exclusion-cidr:
         description: Any packet sent to IP addr belonging to CIDR will not be SNAT'd
         type: string
+  ClockSource:
+    description: |-
+      Status of BPF clock source
+
+      +k8s:deepcopy-gen=true
+    type: object
+    properties:
+      mode:
+        description: Datapath clock source
+        type: string
+        enum:
+        - ktime
+        - jiffies
+      hertz:
+        description: Kernel Hz
+        type: integer
```

Comment:
Having current time would be very useful when debugging e.g. CT issues. Maybe worth adding?